### ISSUE TITLE
Issue: Font, Padding, Design Enhancement, and Dark Mode Implementation

### ISSUE THREAD

**Issue by BhattAnsh** [abhijeetnishal/URLShortener#20]
**### Issue Description**
The current design of the link shortener website requires several improvements to enhance user experience and visual appeal. Specifically, the font needs to be changed, padding should be added to the link submission bar, and the overall design should be made more catchy. Additionally, considering user preferences and accessibility, implementing a dark mode feature would be beneficial.

**### Changes Required**
**Font Change:** Replace the current font with a more modern and legible typeface to improve readability and aesthetics.
Padding Addition: Apply padding to the link submission bar to improve spacing and make it visually distinct.

**Design Enhancement:** Enhance the overall design to make it more visually appealing, engaging, and aligned with the branding or theme of the website.

**Dark Mode Implementation:** Introduce a dark mode feature to provide users with an alternative color scheme that reduces eye strain and improves visibility, especially in low-light environments.

### **Expected Outcome**
Improved readability and aesthetics through font change.
Enhanced user experience with padding addition and overall design enhancements.
Accessibility and user preference catered to with the implementation of dark mode.

**Comment by BhattAnsh** [comment]
Please assign me this issue I can work on this i've worked similar project for frontend.

**Comment by abhijeetnishal** [comment]
Hey @BhattAnsh, There is an issue #15, we are migrating from react to next.js with tailwind CSS, once this issue is merged, we can work on this issue.
I am assigning this issue for now.

**Comment by BhattAnsh** [comment]
@abhijeetnishal so i have to wait till the #15 is solved then i have to work on the issue right?

**Comment by abhijeetnishal** [comment]
Yes if that issue is done then it is easier for you to implement also. 
Till then try to contribute to other issues.

**Comment by abhijeetnishal** [comment]
Hey @BhattAnsh, are you still interested in this issue?

**Comment by BhattAnsh** [comment]
> Hey @BhattAnsh, are you still interested in this issue? 

Yep I am

**Comment by abhijeetnishal** [comment]
I am assigning this issue to you now pull the latest changes from master branch and you are go to go.

**Comment by aaru-shibhardwaj** [comment]
Hello i would like to work on this issue. Please assign me this task

**Comment by me-techshruti54212** [comment]
I would like to work on this issue. So, kindly assign me this project.

**Comment by Anu00009** [comment]
Hello,I would like to work on this issue as i have already done similar type of things in my project .Please assign me this project.

**Comment by BhattAnsh** [comment]
> I am assigning this issue to you now pull the latest changes from master branch and you are go to go.

Can you please transfer it to someone else I can't work right now due to my semester exam datasheet came out.

**Comment by Aish755** [comment]
Hello i would like to work on this issue. Please assign me this task

**Comment by abhijeetnishal** [comment]
I am assigning this task to @aaru-shibhardwaj, she asked first. 
Rest try to work on other issues, Good luck.

**Comment by abhijeetnishal** [comment]
Hey @aaru-shibhardwaj, if possible share the reference UI or flow you are following for this.

**Comment by abhijeetnishal** [comment]
Hey @aaru-shibhardwaj, are you working on this issue?

**Comment by Abhi-0001** [comment]
Hello @abhijeetnishal Can I work on this if this issue is available

**Comment by abhijeetnishal** [comment]
Hey @me-techshruti54212, I am assigning this issue to you now.
